### PR TITLE
Update project to .NET 8

### PR DESCRIPTION
**Pull request by danzhu54**
Contributes to dotnet/website#5444

### CODE REVIEW

File: dotnetthanks-loader.csproj [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="7.0.0" />
```
Comment (mairaw) [inline dotnetthanks-loader.csproj:13]: these packages should also be updated from 7 to 8. Let's look at this right after GA.

**Comment by WeihanLi** [comment]
Maybe we could also update the Dockerfile to .NET 8

**Comment by mairaw** [comment]
Good point @WeihanLi! I'm wondering if we're using that file at all since it hasn't been updated in forever.

**Comment by WeihanLi** [comment]
> I'm wondering if we're using that file at all since it hasn't been updated in forever.

Yeah, from the history, it has not been updated for a long time, and from the search result, the Dockerfile seemed only be used in the vscode settings

https://github.com/search?q=repo%3Adotnet%2Fwebsite-thanks-data%20Dockerfile&type=code

**Comment by danzhu54** [comment]
The docker file was originally used to upload the json file to our resources repo
I think moving that upload step to our scheduled pipeline would be a good next step

**Review by mairaw — APPROVED** [review]

**Pull request merged**